index.js

```diff
@@ -49,63 +49,75 @@ module.exports = {
     }
 
     if (type === 'all') {
-      //
       // Copy our files to assemble our electron and ember apps. Our directory
       // structure inside our build output dir (dist/ or temp directory) is
       // going to end up looking like this:
       //
-      // .
       // ├── package.json
       // ├── .compilerc
-      // ├── <any other files/directories found in ember-electron, except resources-*>
-      // ├── resources
-      // │   ├── <file copied/merged from resources and resources-<platform>>
-      // │   ├── <file copied/merged from resources and resources-<platform>>
-      // │   └── ...
-      // └── ember
-      //     └── <ember build output>
+      // ├── ember
+      // │   ├── <ember build output>
+      // ├── ember-electron
+      //     ├── .electron-forge
+      //     ├── main.js
+      //     ├── resources
+      //         ├── <file copied/merged from resources and resources-<platform>>
+      //         ├── <file copied/merged from resources and resources-<platform>>
       //
-      let funnel = require('broccoli-funnel');
-      let writeFile = require('broccoli-file-creator');
-      let mergeTrees = require('broccoli-merge-trees');
-      let platform = process.env.EMBER_CLI_ELECTRON_BUILD_PLATFORM || process.platform;
+      const funnel = require('broccoli-funnel');
+      const writeFile = require('broccoli-file-creator');
+      const mergeTrees = require('broccoli-merge-trees');
 
-      // Set the main entry point for the electron application, and point the
-      // electron-forge config to ./.electron-forge
       let packageJson = clone(this.project.pkg);
-      packageJson.main = 'lib/index.js';
+      let platform = process.env.EMBER_CLI_ELECTRON_BUILD_PLATFORM
+        || process.platform;
+
+      let emberElectronPath = path.join(this.project.root, 'ember-electron');
+      let compileRcPath = path.join(emberElectronPath, '.compilerc');
```

Comment:
Compilerc is passed to `electron-compile`, and is a required part of forge's upstream packaging flow.